module/criteriaQuestions/helper.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * name : criteriaQuestions.js
+ * author : Aman
+ * created-date : 24-Jun-2020
+ * Description : Criteria questions related information.
+ */
+
+/**
+ * CriteriaQuestionsHelper
+ * @class
+*/
+
+module.exports = class CriteriaQuestionsHelper {
+
+    /**
+   * Create Or update criteria Questions.
+   * @method
+   * @name createOrUpdate
+   * @param {String} criteriaIds - criteria ids.
+   * @param {String} [ updateQuestion = false ] - update question or criteria.
+   * @returns {JSON} - success true or false
+   */
+
+    static createOrUpdate( 
+        criteriaIds,
+        updateQuestion = false
+    ) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let result = "";
+
+                if( Array.isArray(criteriaIds) ) {
+
+                    result = [];
+
+                    for (let criteria = 0; criteria < criteriaIds.length ; criteria++ ) {
+                        
+                        let data = await singleCriteriaCreateOrUpdate(
+                            criteriaIds[criteria]
+                        );
+
+                        result.push({
+                            criteriaId : criteriaIds[criteria],
+                            success : data.success
+                        });
+                    }
+                } else {
+                    result = 
+                    await singleCriteriaCreateOrUpdate(
+                        criteriaIds,
+                        updateQuestion
+                    );
+                }
+
+                return resolve(result);
+
+            } catch (error) {
+                return reject(error);
+            }
+        })
+    }
+    
+};
+
+   /**
+   * Create Or update criteria Questions.
+   * @method
+   * @name singleCriteriaCreateOrUpdate
+   * @param {String} criteriaId - criteria id.
+   * @param {String} updateQuestion - update question or criteria.
+   * @returns {JSON} success true or false
+   */
+
+  function singleCriteriaCreateOrUpdate(criteriaId,updateQuestion) {
+    return new Promise(async function (resolve, reject) {
+        try {
+
+            let criteriaModel = 
+            Object.keys(criteriaSchema.schema);
+
+            let findQuery = {
+                "$match" : {
```

Comment:
@aman-tunerlabs - Didn't we discuss we will put the frameworkCriteriaFilter here ?

Reply:
@akash-above  Yeah Man